server/routes/api/graphql.ts

```diff
@@ -44,23 +46,25 @@ const resolvers: IResolver = {
 			return allUsers.map(userRecordToGraphql);
 		},
 		search_user: async (prev, args) => {
+			const queryRegExp = new RegExp(escapeStringRegexp(args.search.trim()), "i");
```

Comment:
Could searching by regex not be a feature:tm: instead of a bug?

Reply:
What do you mean? Since it's escaped it should be case insensitive non regex search

Reply:
I mean what's the point of escaping possible regex characters in the first place if they can be used to make search more powerful and don't seem to have any security downsides?

Reply:
Oh, I just think it would be confusing for normal people to use. We can add a regex switch